com.github.sharkwouter.minigalaxy.yaml

```diff
@@ -0,0 +1,60 @@
+---
+app-id: com.github.sharkwouter.minigalaxy
+runtime: org.gnome.Platform
+runtime-version: '3.34'
+sdk: org.gnome.Sdk
+
+add-extensions:
+  org.gnome.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    add-ld-path: "."
+    version: "3.34"
+
+  org.gnome.Platform.Compat.i386.Debug:
+    directory: lib/debug/lib/i386-linux-gnu
+    version: "3.34"
+    no-autodownload: true
+
+  org.freedesktop.Platform.GL32:
+    directory: lib/i386-linux-gnu/GL
+    version: "1.4"
+    versions: "19.08;1.4"
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: vulkan/icd.d;glvnd/egl_vendor.d;OpenCL/vendors;lib/dri;lib/d3d;vulkan/explicit_layer.d
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+
+command: minigalaxy
+rename-desktop-file: minigalaxy.desktop
+rename-icon: minigalaxy
```

Comment:
Might be an idea to eventually try and get these renamed upstream too.

Reply:
Also AppData check not passed. Seems like we need to add additional icon size formats to satisfy linter. I don't mind send PR if @sharkwouter will approve this initiative.

Reply:
I would. If it helps, the original source for the icon is one of the jump images in this set: https://opengameart.org/content/jumping-galaxy-asset-cc-by-30

Reply:
@p1u3o what should the image and desktop file be called for these lines to not be needed?

Reply:
> what should the image and desktop file be called for these lines to not be needed?

Should use [reverse-DNS scheme](https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html#sect-Quickstart-DesktopApps). Like
`com.github.sharkwouter.minigalaxy.png`
`com.github.sharkwouter.minigalaxy.desktop`

Reply:
I made some changes upstream now. The desktop, icon and metainfo files have been renamed to use the ``com.github.sharkwouter.minigalaxy`` naming scheme. Both the 128x128 and 192x192 icons are now included.

Reply:
Should I create a tag with these changes to make it easier to build for?

Reply:
> Should I create a tag with these changes to make it easier to build for?

Would be great.

Reply:
Try that flatpak-1 tag